src/main/java/uk/gov/justice/digital/job/S3DataDeletionJob.java

```diff
@@ -60,11 +61,15 @@ private void deleteFiles() {
                 .getConfiguredTables(jobArguments.getConfigKey());
 
         final ImmutableSet<String> bucketsToDeleteFilesFrom = jobArguments.getBucketsToDeleteFilesFrom();
+        final ImmutableSet<String> allowedExtensions = jobArguments.getAllowedS3FileExtensions();
 
         Set<String> failedObjects = new HashSet<>();
 
         for (String bucketToDeleteFilesFrom : bucketsToDeleteFilesFrom) {
-            List<String> objectKeys = new ArrayList<>(s3FileService.listParquetFilesForConfig(bucketToDeleteFilesFrom, configuredTables, 0L));
+            List<String> listedFiles = s3FileService
+                    .listFilesForConfig(bucketToDeleteFilesFrom, configuredTables, allowedExtensions, 0L);
+
+            List<String> objectKeys = new ArrayList<>(listedFiles);
```

Comment:
How come you do `new ArrayList<>(listedFiles)` rather than just use the original list reference?

Reply:
This is now fixed. Thanks 👍